Gopkg.toml

```diff
@@ -151,6 +151,8 @@ ignored = [
   version = "1.12.17"
 
 [[constraint]]
+  # FIXME: this is using a version of boltdb that was patched directly in vendor
+  # to avoid hassle with dep. Make sure to transfer changes when bumping the version
```

Comment:
I ran into issues with teleport when I tried using it from a fork so gave up to free time for main work. Will need to revisit eventually though.

Reply:
What is the issue with bolt / teleport? We should not patch things directly in vendor, that's for sure. Is it hard to fix?